src/core/fair_queue.cc

```diff
@@ -35,6 +35,10 @@
 #include "fmt/format.h"
 #include "fmt/ostream.h"
 
+#if FMT_VERSION >= 90000
+template <> struct fmt::formatter<seastar::fair_queue_ticket> : fmt::ostream_formatter {};
+#endif
+
```

Comment:
I think this should bein fair_queue.hh, next to its `operator<<`. Tickets is public type and sometimes gets printed from e.g. tests (for log-debugging)